tests/plugins/test_checkpoint_io_plugin.py

```diff
@@ -52,7 +52,7 @@ def test_checkpoint_plugin_called(tmpdir):
     )
     trainer.fit(model)
 
-    assert checkpoint_plugin.save_checkpoint.call_count == 5
+    assert checkpoint_plugin.save_checkpoint.call_count == 4
```

Comment:
is this decrease due to the save_last / topk bug mentioned above? but save_last is False by default

Reply:
or because you removed `on_train_end`?

Reply:
The latter if I'm not mistaken.